org.greenfoot.Greenfoot.appdata.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.greenfoot.Greenfoot</id>
+  <name>Greenfoot</name>
+  <developer_name>King's College London</developer_name>
+  <summary>Greenfoot</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-with-classpath-exception</project_license>
+  <url type="homepage">https://greenfoot.org</url>
+  <description><p>Greenfoot is an integrated development environment using Java or Stride designed primarily for educational purposes at the high school and undergraduate level. It allows easy development of two-dimensional graphical applications, such as simulations and interactive games.</p></description>
+  <screenshots>
+    <screenshot type="default">
```

Comment:
These URLs need to be changed to your fork for now, as otherwise validation will fail 404. You can modify them back after the app is accepted.

Reply:
Ok